clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -159,8 +159,11 @@ void printCommands(ArrayRef<StringRef> CmdArgs) {
     return;
 
   llvm::errs() << " \"" << CmdArgs.front() << "\" ";
-  for (auto IC = std::next(CmdArgs.begin()), IE = CmdArgs.end(); IC != IE; ++IC)
-    llvm::errs() << *IC << (std::next(IC) != IE ? " " : "\n");
+  for (auto IC = std::next(CmdArgs.begin()), IE = CmdArgs.end(); IC != IE;
+       ++IC) {
+    llvm::sys::printArg(llvm::errs(), *IC, false);
```

Comment:
can we add an inline comment of what `false` is

Reply:
Sure, added.